tests/playwright-tests/general-public/pages/FindStatisticsPage.ts

```diff
@@ -12,5 +12,6 @@ export default class FindStatisticsPage {
 
   async navigateToPublicReleasePage(publicationName: string) {
     await this.releaseLink(publicationName).click();
+    await this.page.waitForTimeout(2000);
```

Comment:
It might be nice to find another way of handling this; setting manual timeouts is often considered bad practice in UI tests unless absolutely unavoidable because they can be flaky (i.e. this might usually pass, but one day it takes longer than 2 seconds for some reason and fails). They also slow things down unnecessary. The docs discourage them for exactly these reasons too - https://playwright.dev/docs/api/class-page#page-wait-for-timeout 

We should be able to wait for a condition that tells us the page has transitioned, maybe the presence of the new header or something. I believe `getByRole` implicitly waits, so something like this might work:
![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/63285990/a52c27e7-bcfe-4e55-8660-3959dcfe7f13)

Looks like there are a few different ways of doing this in the docs, I think `page.waitForSelector` might work too :)

Reply:
yeah we can do that way as well, but it will break overall consistency  if write await this.page.getByRole('heading', {name:publicationName });